protostar/commands/test/test_execution_environment.py

```diff
@@ -74,6 +73,7 @@ def __init__(
         self._include_paths = include_paths
         self._test_finish_hooks: Set[Callable[[], None]] = set()
         self._starknet_compiler = starknet_compiler
```

Comment:
Aren't `_starknet_compiler` and `_data_transformer` unused?

Reply:
Starknet compiler is used. Data transformer is not.